pom.xml

```diff
@@ -28,7 +28,7 @@
   <packaging>jar</packaging>
   <name>kairosdb</name>
   <description>KairosDB is a fast distributed scalable time series abstraction on top of Cassandra.</description>
-  <version>1.2.3-SNAPSHOT</version>
+  <version>1.2.4-SNAPSHOT</version>
```

Comment:
You should almost never change this by hand. 

The release process will first change this to the desired release version for you (e.g. `1.2.2`). It will tag and commit that triggering a release build on Travis. It will then increment to the desired next development version for you (e.g. `1.2.3-SNAPSHOT`) and commit that for you.